src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -98,6 +96,17 @@ SolverBase::SolverBase( string const & name,
     setRestartFlags( RestartFlags::WRITE_AND_READ ).
     setDescription( "Write matrix, rhs, solution to screen ( = 1) or file ( = 2)." );
 
+  addLogLevel< logInfo::Fields >();
+  addLogLevel< logInfo::LineSearch >();
+  addLogLevel< logInfo::Solution >();
+  addLogLevel< logInfo::Convergence >();
+  addLogLevel< logInfo::TimeStep >();
+  addLogLevel< logInfo::LinearSolver >();
+  addLogLevel< logInfo::NonlinearSolver >();
+  addLogLevel< logInfo::Timers >();
+  addLogLevel< logInfo::Initialization >();
```

Comment:
`logInfo::Initialization` seems to only apply to `SolidMechanicsStateReset` & `PoromechanicsInitialization`.

Reply:
also for statistics